packages/consoledot-api/src/queries/useSSOProvider.ts

```diff
@@ -15,7 +11,8 @@ export const useSSOProvider = () => {
       const response = await fetch(
         `${basePath}/api/kafkas_mgmt/v1/sso_providers`
       );
-      const providers = await response.json();
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const providers: { token_url?: string } = await response.json();
```

Comment:
```suggestion
      const providers = await response.json<{ token_url?: string }>();
```
This should be a bit better, we tell Typescript the shape of the object returned by that json call, so we can do our defensive checks later without having to shut down eslint.

Reply:
Ah, yes sure, I was convinced `fetch` could take a type argument